docs/docs-content/clusters/edge/cluster-management/certificate-renewal.md

```diff
@@ -12,7 +12,9 @@ certificates allows Kubernetes to secure API connections, verify the authenticit
 All certificates have an expiry date, and need to be renewed periodically.
 
 This page guides you through the different methods used to renew certificates in an airgapped Palette Edge cluster. An
-airgapped cluster means a cluster that has no connection to a Palette instance.
+airgapped cluster means a cluster that has no connection to a Palette instance. For information on how to renew
+certificates on connected Edge clusters, refer to
+[Renew Cluster PKI Certificates](../../cluster-management/certificate-management.md).
 
 ## Limitations
```

Comment:
do we need to add the limitation (known issue) that auto cert renewal for connected clusters requires them to be connected? I saw there was a known issue mmentioned by eng. 
Also, what happens if the cluster reconnects at say day t-25 (and hubble tried to do the renewal at t-30), will it then do the renewal?

Reply:
We usually don't add known issues to the limitations section. Limitations are more like expected behavior - they may or may not be removed in the future. But I will clarify regarding when a disconnected Edge host reconnects.